<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifySelfInfo</class>
 <widget class="QDialog" name="ModifySelfInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="title" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(145, 145, 145);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="titleTextLabel">
        <property name="styleSheet">
         <string notr="true">#titleTextLabel{
	font-family:'微软雅黑';
	font-size: 14px;
	font-weight: bold;
	color: #000000;
}</string>
        </property>
        <property name="text">
         <string>修改个人信息</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="body" native="true">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
#body{
	background-color:rgb(239, 239, 239);
}

QPushButton:hover{
	background-color:rgb(59, 59, 59);
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0">
      <property name="leftMargin">
       <number>50</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>50</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>9</number>
      </property>
      <item row="3" column="0">
       <widget class="QLabel" name="nickNameLabel">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>19</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>19</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#nickNameLabel{
	color: #000000;
	font-family: '微软雅黑';
	font-size: 12px;
}</string>
        </property>
        <property name="text">
         <string>昵称nickname</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="3">
       <widget class="QLineEdit" name="nickNameEdit">
        <property name="minimumSize">
         <size>
          <width>258</width>
          <height>38</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>258</width>
          <height>38</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(239, 239, 239);
color: #000000;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="modifyWdBtn">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#modifyWdBtn{
	color: #000000;
	font-family: '微软雅黑';
	font-size: 14px;
	border-radius: 3px;
	border: 2px solid #000000;
}

</string>
        </property>
        <property name="text">
         <string>修改密码</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="pdLabel">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>19</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>19</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#pdLabel{
	color: #000000;
	font-family: '微软雅黑';
	font-size: 12px;
}</string>
        </property>
        <property name="text">
         <string>密码password</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QLineEdit" name="newPassWdEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>258</width>
          <height>38</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>258</width>
          <height>38</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(239, 239, 239);
color: #000000;</string>
        </property>
        <property name="text">
         <string>1234567890</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="cancelBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#cancelBtn{
	border: 1px solid #000000;
	border-radius: 2px;
	text-align:center ;
	padding-left: 10px;
	color: #000000;;
	font-family: '微软雅黑';
	font-size: 14px;
}</string>
        </property>
        <property name="text">
         <string>取消  </string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="phoneLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>34</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>34</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#phoneLabel{
	color: #000000;
	font-family: '微软雅黑';
	font-size: 14px;
}</string>
        </property>
        <property name="text">
         <string>手机号: 19857198709</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="submitBtn">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>40</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>100</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#submitBtn{
	border: 1px solid #000000;
	border-radius: 2px;
	text-align:center ;
	padding-left: 10px;
	color: #000000;
	font-family: '微软雅黑';
	font-size: 14px;
}</string>
        </property>
        <property name="text">
         <string>提交   </string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
